contrail-controller/hooks/contrail_controller_hooks.py

```diff
@@ -143,6 +175,16 @@ def config_changed():
     if config.changed("control-network"):
         ip = common_utils.get_ip()
         settings = {"private-address": ip}
+
+        if config.get('local-rabbitmq-hostname-resolution'):
+            settings.update({
```

Comment:
this 'settings' is for other apps relations. settings for cluster relation is in line 194

Reply:
Thanks for catching - you are right.